src/sumo/wrapper/login.py

```diff
@@ -33,7 +34,7 @@ def get_parser() -> ArgumentParser:
         "--interactive",
         dest="interactive",
         action="store_true",
-        default=False,
+        default=True,
```

Comment:
Will this work? Not sure if it is possible to set this to `False` when action is "store_true". Also, "store_true" (normally?) default to `False`, but here we are overriding which may be confusing?

Possibly need to have a look at the whole design of arguments here...

Reply:
Seems correct. The current config will always set `interactive` to `True` not matter what.

Should either remove `default` completely or set `default=False`